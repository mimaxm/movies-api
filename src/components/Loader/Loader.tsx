import Loader from "react-loader-spinner";

export const DataLoader = () => {
   return(
      <div className="loader">
         <Loader type="ThreeDots" color="#00BFFF" height={80} width={80} />
      </div>
   )
}