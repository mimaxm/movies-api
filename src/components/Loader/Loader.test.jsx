import React from 'react';
import { DataLoader } from './Loader';
import { shallow } from 'enzyme';
import toJson from "enzyme-to-json";

describe('DataLoader testing', () => {
   const component = shallow(<DataLoader />);
   test('shoud have Loader', () => {
      expect(component.find('Loader')).toHaveLength(1)
   });
   test('shoud render without errors', () => {
      expect(toJson(component)).toMatchSnapshot();
   })
})