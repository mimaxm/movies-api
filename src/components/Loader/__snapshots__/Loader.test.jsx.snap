// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataLoader testing DataLoader shoud render without errors 1`] = `
<div
  className="loader"
>
  <Loader
    className=""
    color="#00BFFF"
    height={80}
    style={Object {}}
    timeout={0}
    type="ThreeDots"
    visible={true}
    width={80}
  />
</div>
`;
